sensorimotor/experiments/capacity/defs/vary_elements.yaml

```diff
@@ -0,0 +1,20 @@
+- worlds: 2
```

Comment:
Can you reverse the order of these experiments, so that the fastest, smallest ones run first? Same for `vary_worlds.yml`.